app/src/main/java/org/mozilla/fenix/perf/Performance.kt

```diff
@@ -37,4 +45,59 @@ object Performance {
             activity.reportFullyDrawn()
         }
     }
+
+    /**
+     * Processes intent for Performance testing to remove protection pop up ( but keeps the TP
+     * on) and removes the onboarding screen.
+     */
+    fun processIntentIfPerformanceTest(intent: Intent, context: Context) {
+        if (isPerformanceTest(intent)) {
+            return
+        }
+
+        if (evaluatePhoneStatus(context)) {
```

Comment:
Why not include this as part of `isPerformanceTest`? e.g. `if (isValidPerformanceTest)...`